src/GuestPages/Invite.js

```diff
@@ -16,6 +18,33 @@ const Invite = ({ isPreview, id }) => {
   const backToInviteLink = `/event/${id}/invitation-form`;
   const backtoHomeLink = `/event/${id}/dashboard`;
```

Comment:
It looks good to me. @kyle-sablan  @DylanDKim, please consider if we should hardcode the id since When click on "more edits" or "save changes", it's not redirect correctly. Kyle and I will bring this issue up during our next meeting. 
Approved and ready for the final review